.env

```diff
@@ -0,0 +1 @@
+PORT = 5000
```

Comment:
Do not commit `.env` file. It's meant to store sensitive data like API keys. You should commit `.env.example` instead to specify needed fields.